### ISSUE TITLE
beforeAll inside describe.only triggers no-global-setup

### ISSUE THREAD

**Issue by Moeriki** [tlvince/eslint-plugin-jasmine#145]
**Do you want to request a *feature* or report a *bug*?**

bug

**What is the current behavior?**

```js

describe.only('suite', () => {

  beforeAll(() => { /* localsetup */ }); // 'Do not use `beforeAll` outside a `describe` except for in global helpers.

});
```

**What is the expected behavior?**

No warning.

**Comment by DianaSuvorova** [comment]
Is `describe.only` equivalent to `ddescribe`? Or it is something else?

**Comment by Moeriki** [comment]
Since jest@21 it allows two styles of focus / skip.

`describe.only` is equivalent of `fdescribe` (focus)
`describe.skip` is equivalent of `xdescribe` (skip)

I don't know ddescribe :)

**Comment by DianaSuvorova** [comment]
`ddescribe` is same as `fdescribe` depends on the version of jasmine.
Let me look into `.only` and `.skip` syntax.
